File: MdExplorer/client2/src/app/md-explorer/models/refactoring-filesystem-event.ts
import { IRefactoringFilesystemEvent } from "./irefactoring-filesystem-event";

export class RefactoringFilesystemEvent implements IRefactoringFilesystemEvent{
    Id: string;
    EventName: string;
    NewFullPath: string;
    OldFullPath: string;
    RefactoringGroupId: string;
    Processed: boolean;

}

File: MdExplorer.Abstractions/Models/RefactoringFilesystemEvent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MdExplorer.Abstractions.Models
{
    public class RefactoringFilesystemEvent
    {
        public virtual Guid Id { get; set; }
        public virtual Guid RefactoringGroupId { get; set; }
        public virtual string EventName { get; set; }
        public virtual string OldFullPath { get; set; }
        public virtual string NewFullPath { get; set; }
        public virtual bool Processed { get; set; }
        public virtual RefactoringSourceAction RefactoringSourceAction { get; set; }
    }
}
